tests/integration/test_orders_cli.py

```diff
@@ -791,66 +601,16 @@ def test_cli_orders_request_cloudconfig(invoke, write_to_tmp_json_file):
     assert order_request == json.loads(result.output)
 
 
-@respx.mock
-def test_cli_orders_create_cloudconfig(invoke,
-                                       order_description,
-                                       write_to_tmp_json_file):
-    mock_resp = httpx.Response(HTTPStatus.OK, json=order_description)
-    respx.post(TEST_ORDERS_URL).return_value = mock_resp
-
-    config_json = {
-        'amazon_s3': {
-            'aws_access_key_id': 'aws_access_key_id',
-            'aws_secret_access_key': 'aws_secret_access_key',
-            'bucket': 'bucket',
-            'aws_region': 'aws_region'
-        },
-        'archive_type': 'zip'
-    }
-    config_file = write_to_tmp_json_file(config_json, 'config.json')
-
-    request_result = invoke([
-        'request',
-        '--name',
-        'test',
-        '--id',
-        '4500474_2133707_2021-05-20_2419',
-        '--bundle',
-        'analytic',
-        '--item-type',
-        'PSOrthoTile',
-        '--cloudconfig',
-        config_file
-    ])
-
-    request_file = write_to_tmp_json_file(json.loads(request_result.output),
-                                          'orders.json')
-
-    # Invoke the create call
-    # Invoke the create call
-    result = CliRunner().invoke(cli.main,
-                                ['orders', 'create', str(request_file)],
-                                catch_exceptions=True)
-
-    assert result.exit_code == 0
-    assert json.loads(
-        result.output)['_links']['results'][0]['delivery'] == 'success'
-
-
 def test_cli_orders_request_email(invoke):
     result = invoke([
         'request',
-        '--name',
-        'test',
-        '--id',
-        '4500474_2133707_2021-05-20_2419',
-        '--bundle',
-        'analytic',
-        '--item-type',
-        'PSOrthoTile',
+        '--name=test',
+        '--id=4500474_2133707_2021-05-20_2419',
+        '--bundle=analytic',
+        '--item-type=PSOrthoTile',
         '--email'
     ])
-    assert not result.exception
+    assert result.exit_code == 0
```

Comment:
@mkshah605 @kevinlacaille @sgillies - I really like how checking the exit code is more specific and directly relates to requirements from the interface design doc. I know we have been doing this sporadically - and it is fairly new practice for me - but I just want to highlight this as how we want to do this check going forward.